app/src/main/java/swen_anigans/mathematicfanatic/QuizResultsActivity.java

```diff
@@ -71,14 +71,8 @@ protected void onCreate(Bundle savedInstanceState) {
 
         Star[] starValues = getStars(correct,total,starGroup.getChildCount());
         setStars(starViews,starValues);
-        if(picked == null) {
-            setRewardListeners();
-        } else {
-            LinearLayout slideUp = (LinearLayout) findViewById(R.id.slideuppanel);
-            pickedTV = ((TextView) slideUp.getChildAt(0));
-            slideUp.removeAllViews();
-            formRewardSelectedLayout(slideUp,picked);
-        }
```

Comment:
I do we want the reward to stay selected if the user leaves that activity? If the user goes to the locker they shouldn't be able to go back to that screen. They should have to either A) go through the quiz again to select a new reward or B) go to recess and view what they unlocked.